drivers/console/Kconfig

```diff
@@ -10,6 +10,15 @@
 menuconfig CONSOLE
 	bool "Console drivers"
 
+config UART_CONSOLE_ON_DEV_NAME
+	string "Device Name of UART Device for UART Console"
+	default "$(dt_str_val,DT_UART_CONSOLE_ON_DEV_NAME)" if HAS_DTS
+	default "UART_0"
+	depends on (UART_CONSOLE || CONSOLE_SUBSYS)
+	help
+	  This option specifies the name of UART device to be used for
+	  UART console.
+
```

Comment:
Might want to move this above `menuconfig CONSOLE`. Otherwise, the stuff in the `if CONSOLE` block will no longer appear in a separate menu.

Reply:
Done.